content/en/docs/reference/command-line-tools-reference/kube-apiserver.md

```diff
@@ -344,7 +344,7 @@ kube-apiserver [flags]
 <td colspan="2">--authorization-config string</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>File with Authorization Configuration to configure the authorizer chain.Note: This feature is in Alpha since v1.29.--feature-gate=StructuredAuthorizationConfiguration=true feature flag needs to be set to true for enabling the functionality.This feature is mutually exclusive with the other --authorization-mode and --authorization-webhook-* flags.</p></td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>File with Authorization Configuration to configure the authorizer chain. This feature is mutually exclusive with the other --authorization-mode and --authorization-webhook-* flags.</p></td>
```

Comment:
I think this file is supposed to be generated ... maybe nothing regenerated it after 1.31?